packages/react-scripts-plugin-typescript/src/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const {
+  pushExtensions,
+  pushExclusiveLoader,
+} = require('react-dev-utils/plugins');
+
+function apply(config, { path, paths }) {
```

Comment:
Yeah! Those methods get removed entirely when ejecting -- that package should be the only package that is ever required inside the plugin(s).